### PR TITLE
Simplify PVP configuration

### PR DESCRIPTION
**Pull request by na-ji**
Hello,

I removed the leagues' configuration, since they're fixed. Also, I set default values for pvp level caps. This way, the example configuration is much simpler. 
CF discussion here: https://github.com/UnownHash/Golbat/pull/97#discussion_r1157620260

Thank you.

### CODE REVIEW

**Comment by lenisko** [comment]
This should wait for https://github.com/UnownHash/Golbat/pull/99 as we will get another option in there.

**Comment by jfberry** [comment]
I think you still need the option whether league 500 is little cup rules or not.

**Comment by na-ji** [comment]
> I think you still need the option whether league 500 is little cup rules or not.

Oups, I wasn't aware of that. Which schema do you propose? Just one simple boolean key under pvp?

**Comment by jfberry** [comment]
> > I think you still need the option whether league 500 is little cup rules or not.
> 
> Oups, I wasn't aware of that. Which schema do you propose? Just one simple boolean key under pvp?

Yes, I think thats the way

File: README.md [hunk 1]
```diff
@@ -64,6 +64,16 @@ gyms - process gyms in GMO
...
 cells - process cell updates (disabling this also disables automatic fort clearance)
 
+# PvP
+Extra configurations for PvP are available in the `pvp` section of the config file.
+
+```toml
+# The level caps used in rankings, defaults to [50, 51]
+level_caps = [50, 51]
+# Whether little cup rules should be applied to the 500cp league, default to true
+use_little_cup_rules = true
```
Comment (Fabio1988) [inline README.md:74]: This means now we allow evolutions or not? What's the little cup rules in general? :)

**Comment by Fabio1988** [comment]
@na-ji can you pls resolve conflicts? :)

**Comment by jfberry** [comment]
Is there anything holding this up? This is much better in general and would like to get it merged :-)
I guess we could debate whether the little cup option is named in the best way, but everything else is perfect :-)

**Comment by na-ji** [comment]
> Is there anything holding this up? This is much better in general and would like to get it merged :-)

Nothing is holding this up anymore. I just forgot, even though Fabio reminded me twice, lol. 

> I guess we could debate whether the little cup option is named in the best way, but everything else is perfect :-)

Which name do you suggest?

**Comment by jfberry** [comment]
> little cup option is named in the best way, but everything else is perfect :-)
> 
> Which name do you suggest?

I'm not sure :-)
little_league_supports_evolution ?
just something that makes it more obvious what the option means

**Review by Fabio1988 — APPROVED** [review]
Thanks a lot, now everything should be fine. Lgtm

**Review by jfberry — APPROVED** [review]
lgtm

**Pull request merged**